.pre-commit-config.yaml

```diff
@@ -32,7 +32,7 @@ repos:
   - id: isort
     additional_dependencies: [toml]
 - repo: https://github.com/psf/black
-  rev: 23.1.0
+  rev: 24.4.0
```

Comment:
Is this supposed to be part of these changes?

Reply:
Yes. I was updating the file formatter in this change since a lot of files were moving.
Want me to remove?

Reply:
🙏🏼